tests/unit/builders/indexable-link-builder-test.php

```diff
@@ -22,7 +22,6 @@
  * @group builders
  *
  * @coversDefaultClass \Yoast\WP\SEO\Builders\Indexable_Link_Builder
- * @covers ::<!public>
```

Comment:
Why is this removed and not replaced by `@coversDefaultClass \Yoast\WP\SEO\Builders\Indexable_Link_Builder`